Fst2List.cpp

```diff
@@ -269,6 +269,7 @@ class CFstApp {
   char ofExt[16];
   char ofnameOnly[512];        // output file name
   char defaultIgnoreName[512]; // input file name
+  bool modeMorph = false;  // true if the current state is in morphological mode
```

Comment:
Have you noticed this suggestion https://github.com/UnitexGramLab/unitex-core/pull/82#discussion_r297087253 ?

Reply:
Oops, i forgot this one. Done.